torchvision/datasets/rendered_sst2.py

```diff
@@ -53,18 +54,13 @@ def __init__(
         if not self._check_exists():
             raise RuntimeError("Dataset not found. You can use download=True to download it")
 
-        self._labels = []
-        self._image_files = []
-
-        for p in (self._base_folder / self._split_to_folder[self._split]).glob("**/*.png"):
-            self._labels.append(self.class_to_idx[p.parent.name])
-            self._image_files.append(p)
+        self._samples = make_dataset(str(self._base_folder / self._split_to_folder[self._split]), extensions="png")
```

Comment:
```suggestion
        self._samples = make_dataset(str(self._base_folder / self._split_to_folder[self._split]), extensions=(".png",))
```

Reply:
I believe it ultimately comes down to `endswith` which accepts tuples but also just plain strings.
I think the type annotations are incorrect here.

Reply:
Let me send a PR to fix that.